Dockerfile

```diff
@@ -30,4 +30,4 @@ ENV QUARKUS_DATASOURCE_USERNAME postgres
 ENV QUARKUS_DATASOURCE_PASSWORD postgres
 
 # Start the application
-CMD ["java", "-Xmx15g", "-jar", "agr_mati-runner.jar"]
+CMD ["java", "-Xmx820m", "-jar", "agr_mati-runner.jar"]
```

Comment:
Oops ya this will cause some issues.

Reply:
In what sense? Would quarkus need more memory to operate? I'm sure the mati application as such wouldn't need much memory, as it doesn't need to load file or anything like that.

Reply:
No... I am saying that I am surprised it ran on the server being told to take 15GB.